identifiers/country-ke/county_assembly.csv

```diff
@@ -0,0 +1,1451 @@
+id,name
+ocd-division/country:ke/county:ke01/ed:baringo_central/ward:ewalel_chapchap,Ewalel-Chapchap

```

Comment:
I haven't thoroughly researched this but I have a question about the wards:

Are they really subdivided from the national assembly districts? In other words, if there is a redistricting of the assembly districts in a county, is it valid to have wards move from one assembly district to another?

Reply:
Within the current administrative structure, wards are subdivided from national assembly districts, and the delimitation of electoral units per constitution ought to be reviewed at intervals of roughly between  8  to 12 years depending on population quotas. 

On the uploaded data, some Wards bear similar names due to the fact that these are localized regions and on occasion, names tend to be replicated eg Biashara Ward appears in 3 Constituencies (Ruiru, Naivasha & Nakuru Town East)

Reply:
Thanks for the additional details, but I don't think this answers my question.

Here's a more concrete hypothetical:
Suppose a redistricting happens and the size of Baringo South  `ocd-division/country:ke/county:ke01/ed:baringo_south` increases. With this change in size The number of wards in Baringo South increases from 4 to 5 and the number of wards in Baringo Central decreases from 5 to 4.
Let's suppose Tenges, `ocd-division/country:ke/county:ke01/ed:baringo_central/ward:tenges` is the ward removed from Baringo Central and added to Baringo South.

In this situation should the Tenges Ward be considered the same Ward in the County Government, or would it be considered a new Ward since it has moved to a new electoral district?